app/helpers/users_helper.rb

```diff
@@ -17,6 +17,10 @@ def can_view_funding?
     FeatureService.enabled?("funding") && defined?(current_user) && !current_user&.system_admin
   end
 
+  def can_view_upload_funding?
+    FeatureService.enabled?("upload_funding") && defined?(current_user) && current_user&.system_admin
+  end
+
```

Comment:
can rely on `SystemAdminConstraint` if this moves into the sys admin area

Reply:
sorted